templates/smb_listener/ballerina.conf

```diff
@@ -0,0 +1,9 @@
+SMB_HOST = "192.168.112.8"
```

Comment:
Let's remove the ip and keep it blank? Also, let's keep some default values for the other properties also.

Reply:
Updated.